docs/source/030_installing.rst

```diff
@@ -150,14 +150,18 @@ to get Lmod installed quickly by using the defaults:
   the spider caching system.  This will greatly speed up ``module
   avail`` and ``module spider``
 
-To install Lmod, you'll want to carefully read the following.  If you
-want Lmod version X.Y installed in ``/opt/apps/lmod/X.Y``, just do::
+If you want Lmod version X.Y installed in ``/opt/apps/lmod/X.Y``, use
+the following form of ``configure``::
 
     $ ./configure --prefix=/opt/apps
     $ make install
 
-The ``make install`` will install Lmod in ``/opt/apps/lmod/x.y.z``
-and create a link to ``/opt/apps/lmod/lmod``.
+Take special note!  The ``make install`` will create the ``lmod``
```

Comment:
I tried recasting this so that the paragraph preceding the command explains what will be created.  That may be clearer than putting the explanation after the command.  I think I would catch that now, if I were reading for the first time.